### PR TITLE
fix (curriculum): made the order of the required code to be redundant

### LINKED ISSUE #38311: Changing order of rules, the tests fail [Applied Visual Design: Create Movement Using CSS Animation]
**Describe your problem and how to reproduce it:**
In the Applied Visual Design: Create Movement Using CSS Animation challenge, if you add the left offset property as required by the challenge, but you add it above the existing top offset property, the tests fail

With code
```
<style>
div {
  height: 40px;
  width: 70%;
  background: black;
  margin: 50px auto;
  border-radius: 5px;
  position: relative;
}

#rect {
  animation-name: rainbow;
  animation-duration: 4s;
}

@keyframes rainbow {
  0% {
    background-color: blue;
    left: 0px;
    top: 0px;
    
  }
  50% {
    background-color: green;
    left: 25px;
    top: 50px;
    
  }
  100% {
    background-color: yellow;
    left: -25px;
    top: 0px;
    
  }
}
</style>

<div id="rect"></div>
```
the tests still say
```
// running tests
The @keyframes rule for 0% should use the left offset of 0px.
The @keyframes rule for 50% should use the left offset of 25px.
The @keyframes rule for 100% should use the left offset of -25px.
// tests completed
```

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/create-movement-using-css-animation

**Tell us about your browser and operating system:**
* Browser Name: Google Chrome
* Browser Version: Versione 80.0.3987.122 (Build ufficiale) (a 32 bit)
* Operating System: Windows 10 version 1903

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
Learn Applied Visual Design- Create Movement Using CSS Animation - freeCodeCamp-org 2-3-2020 17-51-00

### PR DESCRIPTION
**Pull request by hassaanp**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38311

Added checks for all possible positions for the required code.

### CODE REVIEW

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Is there no better way we can check this? Directly checking CSS or something like that?

**Comment by RandellDawson** [comment]
@thecodingaviator Yes, I agree.  I started working on incorporating a CSS parser that will avoid us having to write these complicated regex asserts.  The idea is to have a set of global helper function available for us to use in the Tests.  Once such helper lets us check specific CSS values with the parser.

**Comment by thecodingaviator** [comment]
@RandellDawson I have no idea what an ETA on that might be, so do you think we should go ahead with this PR or wait for yours?

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md]: I have tested the below regex that is much simpler.  Please click the `Commit suggestion` if you agree.
```suggestion
    testString: assert(code.match(/50%\s*?{[\s\S]*?left:\s*?25px(;[\s\S]*?|\s*?)}/gi));
```

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md]: I have tested the below regex that is much simpler.  The extra `[^50]` added before the `0%` was to make sure the `50` and the `100` were not caught in the match.  Please click the `Commit suggestion` if you agree.
```suggestion
    testString: assert(code.match(/[^50]0%\s*?{[\s\S]*?left:\s*?0px(;[\s\S]*?|\s*?)}/gi));
```

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md
Comment (RandellDawson) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md]: I have tested the below regex that is much simpler. Please click the Commit suggestion if you agree.
```suggestion
    testString: assert(code.match(/100%\s*?{[\s\S]*?left:\s*?-25px(;[\s\S]*?|\s*?)}/gi));
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@hassaanp @thecodingaviator If we want a more concise regular expression, then you can use my suggestions.  We could use something like this across all such CSS challenges, but the CSS parser I hope to have implemented soon will let us do these checks without regular expression. 

@hassaanp Please hold off on any other changes like this one until I get the CSS parser implemented.

Thanks

**Comment by thecodingaviator** [comment]
@RandellDawson I like your suggestions and if we're looking for a quick fix to this challenge, this should work, however, since it is unlikely these changes would be deployed soon and because the RWD section is under code freeze, I think it best to hold off this PR for now.

cc: @freeCodeCamp/moderators

**Comment by RandellDawson** [comment]
@thecodingaviator The OP can still commit these suggestions.  We just need to wait until the code freeze is over before merging.  Technically, since only the `testString`s are changed, we could probably go ahead an merge this (with @raisedadead's permission), because the changes will not need a translation.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Once all QA is done, let me know or assign me to this I will land this.

**Comment by RandellDawson** [comment]
@hassaanp Please see the suggestions above.

**Comment by hassaanp** [comment]
> @hassaanp Please see the suggestions above.

Thanks for the suggestions. I have applied them.

**Comment by hassaanp** [comment]
@raisedadead please go ahead and merge

**Review by thecodingaviator — APPROVED** [review]

**Pull request merged**
